app/src/main/java/org/eyeseetea/malariacare/layout/utils/LayoutUtils.java

```diff
@@ -161,6 +185,24 @@ public static void setActionbarTitle(ActionBarActivity activity, Spanned title,
         ((CustomTextView) activity.findViewById(R.id.action_bar_multititle_title)).setText(title);
         ((CustomTextView) activity.findViewById(R.id.action_bar_multititle_subtitle)).setText(subtitle);
     }
+    public static void setSurveyActionbarTitle(ActionBarActivity activity, Spanned title,
```

Comment:
this method is used only from own class, can you refactor this method to private?